catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/TestIcebergTable.java

```diff
@@ -459,13 +483,13 @@ public void testAlterIcebergTable() {
         .asTableCatalog()
         .alterTable(
             NameIdentifier.of(tableIdentifier.namespace(), "test_iceberg_table_new"),
-            TableChange.deleteColumn(new String[] {"col_1"}, false));
+            TableChange.deleteColumn(new String[] {"col_3"}, false));
```

Comment:
why change this line?

Reply:
> why change this line?

@FANNG1 i made col_1 as `sort-order` column, and iceberg can not drop sort-order columns.
otherwise, an exception will occur, as follows:
```
Cannot find source column for sort field: bucket[10](1) DESC NULLS FIRST
org.apache.iceberg.exceptions.ValidationException: Cannot find source column for sort field: bucket[10](1) DESC NULLS FIRST
	at org.apache.iceberg.exceptions.ValidationException.check(ValidationException.java:49)
	at org.apache.iceberg.SortOrder.checkCompatibility(SortOrder.java:301)
	at org.apache.iceberg.TableMetadata$Builder.build(TableMetadata.java:1348)
	at org.apache.iceberg.TableMetadata.updateSchema(TableMetadata.java:542)
	at org.apache.iceberg.SchemaUpdate.commit(SchemaUpdate.java:441)
```